pallets/fiat_filter/src/lib.rs

```diff
@@ -148,12 +144,10 @@ impl<T: Config> Module<T> {
         };
         match call.is_sub_type() {
             Some(attest::Call::set_claim(_attester, attestation, _sig)) => {
-                let iri_size: u32 = attestation.iri.as_ref().unwrap_or(&[1].to_vec()).len() as u32;
+                let iri_size_100bytes: u32 =
+                    (attestation.iri.as_ref().unwrap_or(&[1].to_vec()).len() as u32).div_ceil(100);
```

Comment:
This can be simplified as `attestation.iri.as_ref().map_or_else(|| 1, |i| (i.len() as u32).div_ceil(100))`